pkg/scheduler/plugins/cpunormalization/least_allocated.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This file contains code derived from and/or modified from Kubernetes
```

Comment:
If not modified, you can just keep the kubernetes license. and add the file into .licenseignore to skip license-linter.